app/views/stages/_fields.html.erb

```diff
@@ -61,6 +61,17 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
+      <div class="checkbox">
+        <%= form.label :duplicable do %>
+          <%= form.check_box :duplicable %>
+          This stage can serve as a production template.
```

Comment:
I talked to @dragonfax and he had some good ideas, do you like any of the following:

1. Alpha stage (the progenitor stage)
2. Baseline stage
3. Prime stage